.github/workflows/escalation.yml

```diff
@@ -0,0 +1,17 @@
+name: Escalation workflow container image and manifest push
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+    paths:
+      - 'escalation/**'
+      - 'pipeline/**'
+
+jobs:
+  escalation-workflow:
+    uses: ./.github/workflows/main.yml
```

Comment:
in the other e2e we use `uses: parodos-dev/serverless-workflows/.github/workflows/main.yml@main` not sure if that would make a difference

Reply:
it does: you cannot run it from your formed repo (does not inherit the secrets)